main.go

```diff
@@ -82,7 +82,7 @@ func main() {
 		}
 	}
 
-	is, err := imagestore.NewImageStore(isoeditor.NewEditor(Options.DataDir), Options.DataDir, Options.ImageServiceBaseURL, Options.InsecureSkipVerify, versions, Options.AssistedServiceApiTrustedCAFile)
+	is, err := imagestore.NewImageStore(isoeditor.NewEditor(Options.DataDir), Options.DataDir, Options.ImageServiceBaseURL, Options.InsecureSkipVerify, versions, Options.OSImageDownloadTrustedCAFile)
```

Comment:
Should we also pass `AssistedServiceApiTrustedCAFile` when creating the assisted service client?

Reply:
yes, agreed, I have changed this.